frontend/src/scenes/persons/MergePerson.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import api from 'lib/api'
+import { Button, Select } from 'antd'
+import Modal from 'antd/lib/modal/Modal'
+import { PersonType } from '~/types'
+import { useActions, useValues } from 'kea'
+import { personsLogic } from './personsLogic'
+import { toast } from 'react-toastify'
+import posthog from 'posthog-js'
+
+export function MergePersonButton({
+    person,
+    onPersonChange,
+}: {
+    person: PersonType
+    onPersonChange: CallableFunction
+}): JSX.Element {
+    const [isModalOpen, setIsModalOpen] = useState(false)
+    const [selectedPeople, setSelectedPeople] = useState([])
+
+    const { loadPersons, setListFilters } = useActions(personsLogic)
+    const { persons } = useValues(personsLogic)
+    useEffect(() => {
+        loadPersons()
+    }, [])
+
+    useEffect(() => {
+        loadPersons()
+    }, [person.distinct_ids, person.id])
```

Comment:
How come there are two `useEffects` that do the same thing? Even with the dependency array I believe this should still run on render